Lib/pathlib.py

```diff
@@ -640,7 +640,7 @@ def with_suffix(self, suffix):
         return self._from_parsed_parts(self._drv, self._root,
                                        self._parts[:-1] + [name])
 
-    def relative_to(self, *other):
+    def relative_to(self, other, *args):
```

Comment:
Maybe name this something scarier so that people who use a fancy editor don't wonder what `args` means? And since this is changing, we might as well make it positional-only since the single argument doesn't make a lot of sense to be passed by keyword.

If this makes sense then we should apply this to `is_relative_to()` as well.


```suggestion
    def relative_to(self, other, /, *_deprecated):
```